### ISSUE TITLE
What's the state of this repo

### ISSUE THREAD

**Issue by flotho** [sbidoul/typodoo#4]
Hi @sbidoul ,

Thanks for this awesome work, I've discovered your talk in Odooexp22 and it was awesome!
Is this repo the best place to test, share and report issues?
Regards

**Comment by sbidoul** [comment]
@flotho you can test with this repo, together with this Odoo patch: https://github.com/odoo/odoo/pull/103227. We recently rebased it on the latest 16.0.

Of course to get the full benefits, the Odoo ORM patch needs to be elaborated a little bit, and then it must percolate to official addons. Maybe @rco-odoo can give an update about Odoo plans about this.

**Comment by rvalyi** [comment]
one more question: it seems we can provide type checker some extra type declaration stubs files. Seems this is used for type inference with legacy libraries.

What about generating such stubs files in OCA/OCB automatically untill odoo/odoo start integrating the new field syntax?

**Comment by sbidoul** [comment]
@rvalyi yes, many things can be done. For instance it should be feasible and not *that* hard to write a script that automatically convert the inheritance declarations. So there are way to accelerate the conversions. Creating typing stubs for models.py fields.py and api.py may be feasible, but I'm not sure it's feasible for the official addons.
